packages/@n8n/db/src/entities/user.ts

```diff
@@ -53,9 +53,9 @@ export class User extends WithTimestamps implements IUser, AuthPrincipal {
 	@Length(1, 32, { message: 'Last name must be $constraint1 to $constraint2 characters long.' })
 	lastName: string;
 
-	@Column({ nullable: true })
+	@Column({ type: String, nullable: true })
```

Comment:
Did typeorm require `type: String` when you updated here? In others I see `type: 'varchar'` and in the DB schema it's `VARCHAR `.

I was wondering if we don't need to specify `type: String` then maybe be better not to add it, so one less spot that can get out of sync.

Reply:
We need to specify a type for typeorm to allow string | null, I used the String type, because mfaSecret was doing it the same way.